README.md

```diff
@@ -36,13 +36,108 @@ Examples:
   validate-ddf ../ddf-example --exclude-tags "WARNING_TAG"           Get all kinds of issues except warnings
 ```
 
-### index file creation
+## API usage
+
+First of all you should install this package: `npm i ddf-validation`
+
+API for `ddf-validation` has one method that expects particular type of validator and 3 types of validator:
+
+ * JSON based validator (`JSONValidator`)
+ * stream based validator (`StreamValidator`)
+ * validator that returns only kind of current DDF dataset: `correct/true` or `incorrect/false`
```

Comment:
Validator that checks whether dataset has errors and if there are any - returns `true`, otherwise - `false`